extensions/common/context.cc

```diff
@@ -170,6 +170,8 @@ void populateRequestInfo(bool outbound, bool use_host_header_fallback,
   uint64_t response_flags = 0;
   getValue({"response", "flags"}, &response_flags);
   request_info->response_flag = parseResponseFlag(response_flags);
+
+  getValue({"connection_id"}, &request_info->connection_id);
```

Comment:
Shall we put this in extendedRequestInfo?

Reply:
Thanks..done